src/registrar/templates/includes/domains_table.html

```diff
@@ -53,7 +73,24 @@ <h2 id="domains-header" class="display-inline-block">Domains</h2>
     </div>
     {% endif %}
   </div>
-  {% if portfolio %}
+
+  <!-- Banner for when it's not org model bc placement in the table -->
```

Comment:
Condense this to non-org model banner

Reply:
Open to feedback on editing this comment as well, but I left this comment as a "just in case" for ppl wondering why it looks duplicated and why it's placed in this section 😅

Reply:
Same comment as above, let's at least condense the beginning part to Non org model banner for readability